Sources/Orbit/Components/InputField.swift

```diff
@@ -255,7 +342,34 @@ struct InputFieldLivePreviews: PreviewProvider {
             }
             .animation(.easeOut(duration: 0.25), value: message)
             .padding()
-            .previewDisplayName("Run Live Preview")
+            .previewDisplayName("Run Live Preview with Input Field")
+        }
+    }
+
+
+    struct PreviewWrapperSecured: View {
+        @State var message: MessageType = .none
```

Comment:
Does the message change at all? It seems to be not bound to anything. A `PreviewWrapperWithState` could be used instead to handle the `value`.